src/openvic-simulation/politics/Government.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "openvic-simulation/types/IdentifierRegistry.hpp"
+#include "openvic-simulation/politics/Ideology.hpp"
+
+namespace OpenVic {
+    struct GovernmentTypeManager;
+
+    struct GovernmentType : HasIdentifier {
+        friend struct GovernmentTypeManager;
+
+    private:
+        std::vector<std::string> ideologies;
+        const bool elections, appoint_ruling_party;
+        const uint16_t election_duration;
+        const std::string_view flag_type_identifier;
```

Comment:
```suggestion
        const std::string flag_type_identifier;
```
This needs to store its own copy of the data - the `std::string_view` which is (correctly) in the constructor is only valid until the end of the load function.